adam/curriculum/verbs_with_dynamic_prepositions_curriculum.py

```diff
@@ -132,6 +134,7 @@ def _push_to_template(
                 during=DuringAction(continuously=[on(theme, surface)]),
             )
         ],
+        after_action_relations=[near(theme, goal_reference)],
```

Comment:
Is there a reason why you needed to add this relationship?

Reply:
In Chinese, there are two different coverbs that are used depending on whether a motion ends in a static location or not and right now we use after-action relations to indicate when it doesn't (https://github.com/isi-vista/adam/issues/796)